acorn/dist/acorn.d.ts

```diff
@@ -36,8 +38,44 @@ declare namespace acorn {
   }
 
   class Parser {
+    // state.js
+    lineStart: number;
+    options: Options;
+    curLine: number;
+    start: number;
+    end: number;
+    input: string;
+    type: TokenType;
+
+    // state.js
     constructor(options: Options, input: string, startPos?: number)
     parse(this: Parser): Node
+
+    // tokenize.js
+    next(): void;
+    nextToken(): void;
+
+    // statement.js
+    parseTopLevel(node: Node): Node;
+
+    // node.js
+    finishNode(node: Node, type: string): Node;
+    finishNodeAt(node: Node, type: string, pos: number, loc: Position): Node;
+
+    // location.js
+    raise(pos: number, message: string) : void;
+    raiseRecoverable?(pos: number, message: string) : void;
+
+    // parseutils.js
+    unexpected(pos: number) : void;
+
+    // index.js
+    static acorn: {
```

Comment:
It seems like `static acorn: typeof acorn` would work here to automatically include all exports.